### PR TITLE
closes #1575

### LINKED ISSUE #1575: Fix file upload for avatar and cert documents on user edit

### PR DESCRIPTION
**Pull request by kmanzana**

### CODE REVIEW

File: app/views/users/_certification_fields.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<fieldset>
+  <div class="col-md-11">
+    <%= f.select :certification_name_id,
+      options_from_collection_for_select(CertificationName.same_account_as(current_user), :id, :name, f.object.certification_name_id) %>
+
+    <div class="date-select">
+     <%= f.date_select :issue_date, { start_year: Time.now.year+3,
```
Comment (duffcodester) [inline app/views/users/_certification_fields.html.erb]: why is start year in the future and end year in the past?

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -62,10 +62,12 @@ def edit
 
   def update
     if @user.update_attributes(user_params)
+      flash[:info] = 'Employee Profile updated'
+
       mixpanel_track_people
       sign_in(@user, bypass: true) if @user == current_user
       Tracker.track(current_user.id, 'Update User Profile')
-      render json: @user.to_json(include: [:certifications])
+      redirect_to @user
```
Comment (duffcodester) [inline app/controllers/admin/users_controller.rb]: do we need to include the certifications or does the controller have them hard-code included?

File: app/views/users/edit.html.erb [hunk 3]
```diff
@@ -1,135 +1,100 @@
...
-      <dq-form-field field='user.city' label='City'></dq-form-field>
-      <label>State</label>
-      <select ng-model='user.state' class='form-control'
-        ng-options='state as state for state in controller.usStates'>
-      </select>
-      <dq-form-field field='user.zipcode' label='Zipcode'></dq-form-field>
-      <!-- NEED TO FIX <dq-form-field field='user.avatar' label='Avatar' type='file'></dq-form-field> -->
-    </div>
-    <br>
+      <%= f.file_field :avatar, label: 'User Avatar' %>
```
Comment (duffcodester) [inline app/views/users/edit.html.erb]: when I tried to upload a file and clicked update I got a missing template user update error

Comment (kmanzana) [inline app/views/users/edit.html.erb]: should be seeing errors now on the form

**Comment by michaelDpierce** [comment]
Good on staging :+1:

**Pull request merged**